ui/app/ducks/swaps/swaps.js

```diff
@@ -38,7 +42,7 @@ import {
   SWAP_FAILED_ERROR,
 } from '../../helpers/constants/swaps'
 import { SWAP, SWAP_APPROVAL } from '../../helpers/constants/transactions'
-import { fetchBasicGasAndTimeEstimates, fetchGasEstimates, resetCustomGasState } from '../gas/gas.duck'
+import { resetCustomGasState } from '../gas/gas.duck'
```

Comment:
We shouldn't need to reset the custom gas state anymore if we're not using the custom gas state in swaps.

Reply:
Addressed in a71334840